filament/src/RenderPass.cpp

```diff
@@ -590,10 +589,10 @@ RenderPass::Command* RenderPass::generateCommandsImpl(RenderPass::CommandTypeFla
         // soaInstanceInfo[i].count is the number of instances the user has requested, either for
         // manual or hybrid instancing. Instanced stereo multiplies the number of instances by the
         // eye count.
-        if (UTILS_UNLIKELY(hasStereo)) {
+        if (UTILS_UNLIKELY(hasInstancedStereo)) {
             cmdColor.primitive.instanceCount =
-                    (soaInstanceInfo[i].count * stereoEyeCount) |
-                    PrimitiveInfo::USER_INSTANCE_MASK;
+                (soaInstanceInfo[i].count * stereoEyeCount) |
+                PrimitiveInfo::USER_INSTANCE_MASK;
```

Comment:
nit: we use a continuation indent of 8 spaces

Reply:
Done